cmd/lakectl/cmd/root_test.go

```diff
@@ -71,58 +71,45 @@ func TestLakectlUnixPerm(t *testing.T) {
 }
 
 func TestInitConfig_LoadingScenarios(t *testing.T) {
-	type setupFn func(t *testing.T) (cleanup func())
-
 	tests := []struct {
 		name        string
-		setup       setupFn
+		setup       func(t *testing.T)
 		expectedURL string
 	}{
 		{
 			name: "using --config flag",
```

Comment:
relevant to all tests: the name is also used in the case you like to run it later - suggest to keep them simple. example: file, home, env, file_and_env